src/Impl/RestPublisher.php

```diff
@@ -30,12 +31,12 @@ public function __construct(
     ) {
         $this->headers = $this->buildHttpHeaders($authToken);
         $this->messageIngestionEndpointUri = $this->buildMessageIngestionUri($ingestionEndpointUri, $channel);
-        $this->payloadTypeName = $this->convertToPayloadTypeName($channelSchemaName);
+        $this->payloadTypeName = self::convertToPayloadTypeName($channelSchemaName);
         $this->httpRequester = $httpRequester;
         $this->jsonProvider = $jsonProvider;
     }
 
-    private function convertToPayloadTypeName($channelSchemaName): string
+    private static function convertToPayloadTypeName($channelSchemaName): string
```

Comment:
remain of other development that was itself reverted, but this method still can be static as an utility stateless method